website/docs/podman/gpu.md

```diff
@@ -210,6 +212,21 @@ $ nvidia-ctk cdi list
 
 More information as well as troubleshooting tips can be found [on the official NVIDIA CDI guide](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/cdi-support.html).
 
+4. If on a SELinux-enabled OS (e.g. Fedora family), configure SELinux policy:
```

Comment:
```suggestion
4. Configure the SELinux policy on a SELinux-enabled OS, such as OSs from the Fedora family:
```

Reply:
I did not want to use the "optional" wording because it can potentially convey the wrong message. The user may see this as "something that will improve the experience but not strictly necessary", but that's not the case. If you don't do this on SELinux-enabled systems, your GPU won't work.

Reply:
Thanks for the clarification. I mentioned Optional because if the user is not on any SELinux-enabled OS, then this step is not necessary for them. But, i agree it might convey a wrong message. What about Conditional, if the user is on a SELinux-enabled OS.

Reply:
4. Conditional: On a SELinux-enabled OS, such as OSs from the Fedora family, configure the SELinux policy:

Reply:
Perhaps it's best to keep the heading simple and talk about the conditional stuff on the first line.

```suggestion
4. Configure SELinux (if applicable)

On SELinux-enabled OSes, such as OSs from the Fedora family, the default policy usually disallows containers to have direct access to devices. We make sure it's allowed.
```